alert-policies/amazon-database-migration-service/LowFreeableMemory.yml

```diff
@@ -0,0 +1,36 @@
+name: Low Freeable Memory
+
+description: |+
+  This alert occurs when Freeable Memory bytes go below 100 for 5 minutes
+
+type: STATIC
+
+nrql:
+  query: "SELECT average(aws.dms.FreeableMemory) from Metric"
+
+# Function used to aggregate the NRQL query value(s) for comparison to the terms.threshold (Default: SINGLE_VALUE)
+valueFunction: SINGLE_VALUE 
+
+# List of Critical and Warning thresholds for the condition
+terms:
+  - priority: CRITICAL
+    # Operator used to compare against the threshold.
+    operator: BELOW
+    # Value that triggers a violation; float value
+    threshold: 100
```

Comment:
Please specify the unit of measurement.

Reply:
Done. Unit specified in comment. Please review the changes.